src/executors/ParallelExecutor.js

```diff
@@ -1,7 +1,20 @@
 import ExecutorAbstract from '../ExecutorAbstract';
 
 class ParallelExecutor extends ExecutorAbstract {
+  constructor(thread, executorArray) {
+    super(thread);
 
+    this.executorArray = executorArray;
+  }
+
+  _execute(doneHandler, failHandler) {
```

Comment:
No reason to have this method if we are just passing stuff through to its super.